docs/source/reference/tabular_classification.rst

```diff
@@ -35,8 +35,8 @@ We can use the Flash Tabular classification task to predict the probability a pa
 We can create :class:`~flash.tabular.TabularData` from csv files using the :func:`~flash.tabular.TabularData.from_csv` method. We will pass in:
 
 * **train_csv**- csv file containing the training data converted to a Pandas DataFrame
-* **categorical_input**- a list of the names of columns that contain categorical data (strings or integers)
-* **numerical_input**- a list of the names of columns that contain numerical continuous data (floats)
+* **cat_cols**- a list of the names of columns that contain categorical data (strings or integers)
+* **num_cols**- a list of the names of columns that contain numerical continuous data (floats)
```

Comment:
why not keep `categorical_` and `numerical_` when we have extremely long names elsewhere

Reply:
+1 on this.

Reply:
+1

Reply:
Better to start reducing it, and there were inconsistency inside the code.

Reply:
what does it mean "Better to start reducing it"? to make it shorter? well, it was you who added new long names in the very last PR...